snippets/csharp/VS_Snippets_CLR_System/system.String.IndexOf/CS/ignorable24.cs

```diff
@@ -3,21 +3,23 @@
 
```

Comment:
@WilliamAntonRohm I've removed the changes here because they depend on culture, so we shouldn't touch those. Make sure that the changes in dotnet-api-docs are also reverted.